projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -517,6 +517,10 @@ public function get_profile_link( $service_name, $connection ) {
 			return 'https://twitter.com/' . substr( $cmeta['external_display'], 1 ); // Has a leading '@'.
 		}
 
+		if ( 'bluesky' === $service_name ) {
+			return 'https://bsky.app/profile/' . $cmeta['external_id'];
```

Comment:
Can we use `external_name` here instead?

Reply:
Wouldn't that be the profiel name if you set up a display name? If so then that would not work

Reply:
No, `external_name` for now is always the handle.

Reply:
I see! Do you think we should change to that or keep the Id?

Reply:
I think both are fine. Let us live with ID for now.